ckanext/datagovtheme/tests/test_datagovtheme.py

```diff
@@ -36,6 +38,12 @@ def test_datagovtheme_css(self):
         app = self._get_test_app()
 
         index_response = app.get('/dataset')
+        
+        if ckanext.datagovtheme.helpers.is_bootstrap2():
```

Comment:
You should mock `is_bootstrap2` with `mock`'s `return_value`. This should be two separate tests. In one: "given that is_bootstrap2 returns false, assert datagovtheme.css is in the response body and datagovtheme_boostrap2.css is not in the reponse body"
and
"given that is_bootstrap2 returns true, assert that datagovtheme_bootstrap2.css is in the response body".

Reply:
I can't mock the is_bootstrap2 method in the way you are asking but I feel like this test does handle the case of 

"given that is_bootstrap2 returns true, assert that datagovtheme_bootstrap2.css is in the response body"